<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddModelWindow</class>
 <widget class="QDialog" name="AddModelWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background: rgba(50, 50, 50, 255);
}</string>
  </property>
  <widget class="QLabel" name="label_0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel{
    color: rgba(200, 200, 200, 180);
}</string>
   </property>
   <property name="text">
    <string>Название файла:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_file">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_file">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: rgba(200, 200, 200, 180);
    border-radius: 5px;
    background: rgba(20, 20, 20, 180);
    }

QPushButton:hover {
    background: rgba(10, 10, 10, 180);
    }

QPushButton:pressed {
    border-style: inset;
    background: rgba(0, 0, 0, 180);
    }</string>
   </property>
   <property name="text">
    <string>Choose file</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_moveX">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel{
    color: rgba(200, 200, 200, 180);
}</string>
   </property>
   <property name="text">
    <string>Смещение:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel{
    color: rgba(200, 200, 200, 180);
}</string>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel{
    color: rgba(200, 200, 200, 180);
}</string>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_moveY">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel{
    color: rgba(200, 200, 200, 180);
}</string>
   </property>
   <property name="text">
    <string>Z:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_moveZ">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel{
    color: rgba(200, 200, 200, 180);
}</string>
   </property>
   <property name="text">
    <string> Масштабирование:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel{
    color: rgba(200, 200, 200, 180);
}</string>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_scaleX">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel{
    color: rgba(200, 200, 200, 180);
}</string>
   </property>
   <property name="text">
    <string>Z:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel{
    color: rgba(200, 200, 200, 180);
}</string>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_scaleZ">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_scaleY">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: rgba(20, 20, 20, 180);
	color: rgba(200, 200, 200, 255);
	border-style: inset;
    border-radius: 5px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Цвет:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: rgba(200, 200, 200, 180);
    border-radius: 5px;
    background: rgba(20, 20, 20, 180);
    }

QPushButton:hover {
    background: rgba(10, 10, 10, 180);
    }

QPushButton:pressed {
    border-style: inset;
    background: rgba(0, 0, 0, 180);
    }</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: rgba(200, 200, 200, 180);
    border-radius: 5px;
    background: rgba(20, 20, 20, 180);
    }

QPushButton:hover {
    background: rgba(10, 10, 10, 180);
    }

QPushButton:pressed {
    border-style: inset;
    background: rgba(0, 0, 0, 180);
    }</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_color">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: rgba(200, 200, 200, 180);
    border-radius: 5px;
    background: rgba(20, 20, 20, 180);
    }

QPushButton:hover {
    background: rgba(10, 10, 10, 180);
    }

QPushButton:pressed {
    border-style: inset;
    background: rgba(0, 0, 0, 180);
    }</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
